tests/test_sql_validator.py

```diff
@@ -141,12 +141,28 @@ def test_extract_error_details_error_non_str_message_details(validator, project)
         validator._extract_error_details(query_result)
 
 
-def test_extract_error_details_error_loc_wo_msg_details(validator, project):
-    message = "An error message."
-    sql = "SELECT * FROM orders"
+def test_extract_error_details_for_unset_access_filter(validator, project):
+    message = (
+        'Access filter field "foo.bar" on model "baz.qux" '
+        + "is unset for the current user."
+    )
+    sql = None
     query_result = {
         "status": "error",
-        "data": {"errors": [{"message": message}], "sql": sql},
+        "data": {
+            "errors": [
+                {
+                    "message": message,
+                    "message_details": None,
+                    "params": None,
```

Comment:
Can you remove these other response keys (`params`, `edit_url`, etc.) unless they're being used in the test? Goal is to make this an abstract representation of exactly what's being tested, not the verbatim response that generated the problem being fixed here!